### PR TITLE
Rubocop: Site Drop

### PR DESCRIPTION
**Pull request by pathawks**
#4885

### CODE REVIEW

**Comment by envygeeks** [comment]
Travis doesn't like this pull but it's wrong, nothing changed here other than style.  :shipit:

File: .rubocop.yml [hunk 1]
```diff
@@ -14,8 +14,6 @@ AllCops:
   - lib/jekyll/convertible.rb
   - lib/jekyll/deprecator.rb
   - lib/jekyll/document.rb
-  - lib/jekyll/drops/site_drop.rb
-  - lib/jekyll/entry_filter.rb
```
Comment (parkr) [inline .rubocop.yml]: Remove this – `entry_filter` is where most of the problems are. 😄

**Comment by parkr** [comment]
@pathawks You'll also need to rebase to make that `drop` rubocop issue go away 😄

**Comment by pathawks** [comment]
> Remove this – entry_filter is where most of the problems are.

DOH! :feels good::+1:

**Comment by pathawks** [comment]
Does @jekyllbot notice when a PR has been rebased? That'd be cool

**Comment by parkr** [comment]
> Does @jekyllbot notice when a PR has been rebased? That'd be cool

Not at the moment. The bot [gets a PullRequestEvent](https://developer.github.com/v3/activity/events/types/#pullrequestevent) with `action` = `synchronize`. I took a look yesterday but so far didn't see anything about a force push vs a different kind of push. It's not a PushEvent because you're pushing to a different repo technically. I'll ask about this.

**Comment by pathawks** [comment]
Well, it has been rebased, whether or not @jekyllbot is willing to admit it 😜

**Comment by envygeeks** [comment]
@parkr it should be able to double back onto the status API and see whether or not it conflicted.

**Comment by parkr** [comment]
@jekyllbot: merge +dev

**Pull request merged**